packages/block-directory/src/store/actions.js

```diff
@@ -66,14 +66,15 @@ export function* installBlockType( block ) {
 			throw new Error( __( 'Block has no assets.' ) );
 		}
 		yield setIsInstalling( block.id, true );
-		yield apiFetch( {
-			path: '__experimental/block-directory/install',
+		const response = yield apiFetch( {
+			path: '__experimental/plugins',
 			data: {
 				slug: block.id,
+				status: 'active',
 			},
 			method: 'POST',
 		} );
-		yield addInstalledBlockType( block );
+		yield addInstalledBlockType( { ...block, plugin: response.plugin } );
```

Comment:
This is a bit of a workaround, since we need the main plugin file for uninstallation (ex, `blissful-buttons/blissful-buttons.php`, or `wp-p5js-block/plugin.php`). I'm not sure if we can/should try adding this to the block directory response (from wp.org), or leave this as-is.

Reply:
I don't think the block directory response could return that info nor should it.

I'm fine with this approach for now, but since `plugin` is a pretty generic property we could run into collisions in the future.